### PR TITLE
Feature/dops 1111

### PR DESCRIPTION
**Pull request by Mateorus**
Implement Jenkins-library.
Change naming for project.
Tweak gradle settings to get it work.

### CODE REVIEW

File: Jenkinsfile
Comment (BulatSaif) [inline Jenkinsfile]:
```suggestion
  registry: 'docker.soramitsu.co.jp',
  nexusUserId: "bot-sora-rw"
```

add `bot-sora-rw` to jenkins: https://jenkins.soramitsu.co.jp/job/sora/credentials/

**Comment by BulatSaif** [comment]
I tested the pushing stage and it failed, you can also test it by temporarily adding  in Jenkinsfile:
```
  registry: 'docker.soramitsu.co.jp',
  pushTags: ['PR-231': 'test-ci']
```

File: Jenkinsfile
Comment (Mateorus) [reply Jenkinsfile]: Done

**Review by rkharisov — APPROVED** [review]

File: build.gradle
Comment (BulatSaif) [inline build.gradle]:
```suggestion
    docker {
        // docker tag
        tag = System.getenv("DOCKER_TAG")
        // jar file that is used in the generated Dockerfile
        jar = new File("build/libs/${project.name}-${project.version}.jar")
    }
```
`registry` - it is the default behavior, you can remove it 
add `tag` the imige is pusged with wrong tag, should be `test-ci`

Screenshot from 2021-03-05 19-26-19

Comment (Mateorus) [reply build.gradle]: That was my thought at first, but seems like not. Also, that code added @rkharisov.
Screen Shot 2021-03-05 at 19 55 49

Comment (rkharisov) [reply build.gradle]: `registry` declaration can be safely removed from build script. Keeping it is not a mistake, but just unnecessary due it will be overridden with env variables.

Comment (Mateorus) [reply build.gradle]: Ok. @BulatSaif Should I remove image with wrong tag from harbor?

Comment (BulatSaif) [reply build.gradle]: > Ok. @BulatSaif Should I remove image with wrong tag from harbor?

yes, but wait until you merge this pr, you will need to check harbor again: check that `dev` tag is present and delete `test-ci` and wrong tag.

**Review by Cre-eD — COMMENTED** [review]
It's better to configure Sonarqube before merging

File: build.gradle
Comment (Mingela) [inline build.gradle]: is it needed?

Comment (Cre-eD) [reply build.gradle]: removed

**Review by Cre-eD — APPROVED** [review]
Seems we can't get coverage, so it will be done in a separate task, you can merge it

**Pull request merged**
